jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -266,20 +261,19 @@ def search(self, error_on_fail=False):
             self.table.headers_avail = self.headers + [
                 col for col in column_names if col not in self.headers]
             self.table.headers_visible = self.headers
-            self.app._catalog_source_table = table
             if len(table['sky_centroid']) > self.max_sources:
-                skycoord_table = table['sky_centroid'][:self.max_sources]
+                self.app._catalog_source_table = table[:self.max_sources]
```

Comment:
Looks like max_sources is something user can change. What if user set max_sources to 10, and then to 100? This line overwrites `self.app._catalog_source_table` with max_sources right off the bat. Is the app caching the full table somewhere else or have to re-query?

![Screenshot 2025-02-26 115626](https://github.com/user-attachments/assets/6e4f56b4-199d-499a-97e6-17b448430975)

Reply:
The user has to re-query if they want more sources.